fraud/interface.go

```diff
@@ -50,3 +51,9 @@ type Subscription interface {
 	Proof(context.Context) (Proof, error)
 	Cancel()
 }
+
+// Syncer encompasses the behavior to get fraud proofs from external peers.
+type Syncer interface {
+	// Sync explicitly requests and validates Fraud Proofs from other nodes.
+	Sync(context.Context)
+}
```

Comment:
I am super skeptical that this is worth a method:
* We do not need to enable or disable syncing in the runtime.
	* Even if we would do so, then there should be respective StopSync or similar.
* The only place it is used is in the Availability constructor, meaning in Node build time. If we only need it for build time, then it would be much easier to either:
	* Pass the bool into the FraudService constructor that enables or disables client or requesting side 
	* Or have two implementations of FraudService where one implements server side and another one embeds server side and has an additional client/ requesting side.

Reply:
removed